### PR TITLE
Feature/user read email message any 164595402

### PR DESCRIPTION
**Pull request by Ivankivu**
This PR updates develop branch and fix the following bugs:
- clear password suggestion in the sign in the input field.
- remove default form focus colour

- make the sign in or signup form transparent

- on login, change the background image

- disable hover on reading messages

- make the logo a home link

### CODE REVIEW

File: UI/admin.html [hunk 1]
```diff
@@ -8,255 +8,101 @@
...
-                                       </tbody>
-                               <tfoot>
-                                       <tr><td><input type="submit" value="Save Group" id="sendinmail" onclick="myFunc()"></form></td></tr>
-                               </tfoot>
-
+                <tbody>
+                    <tr>
+                        <td>
+                            <form name="form" onsubmit="required()">
+
```
Comment (mendozabree) [inline UI/admin.html]: Please remove this blank line

File: UI/css/main.css [hunk 2]
```diff
@@ -169,36 +176,68 @@ input[type="password"]:hover{
...
+.container3>.box2>form>.signup {
+    background-color:blue;
+    border:none;
+    margin-bottom: 20px;
+}
+
 .box2>form>div>input:hover {
-    border-color:blue;
+    border-color: blue;
 }
```
Comment (mendozabree) [inline UI/css/main.css]: Remove the blank lines from this file

File: UI/css/style.css [hunk 3]
```diff
@@ -1,53 +1,36 @@
...
-    margin-bottom:10px;
-}
+.container4 {
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+  padding: 0;
+  background-size: 100%;
+  height: 100%;
+  padding: 16px;
```
Comment (mendozabree) [inline UI/css/style.css:9]: Remove blank lines from this file

File: UI/index.html [hunk 4]
```diff
@@ -12,48 +12,39 @@
 
 <body>
```
Comment (mendozabree) [inline UI/index.html]: Remove blank lines from this file

File: UI/signin.html [hunk 5]
```diff
@@ -7,47 +7,40 @@
     <title>EPIC Mail-Sign in</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" type="text/css" media="screen" href="css/main.css">
-    <script src="js/main.js"></script>
```
Comment (mendozabree) [inline UI/signin.html:10]: Remove blank lines from this file

File: UI/user.html [hunk 6]
```diff
@@ -241,16 +242,25 @@ <h1>EPIC Mail</h1>
...
     <script src="js/main.js"></script>
-
     <script>
-    function rowHide(){
-        var x = document.getElementById("rowHidden");
-        if (x.style.display === "none"){
-            x.style.display = "block";
-        } else {
-            x.style.display = "none";
+        function rowHide() {
```
Comment (mendozabree) [inline UI/user.html]: Is it possible to move this `js`  to a js file?
By this, I mean all the js functions here

**Review by mendozabree — CHANGES_REQUESTED** [review]
Nice work @Ivankivu, I like the new look and I see you implemented most of the feedback. 
Please work on this feedback and let me know when you're done.

File: UI/admin.html (hunk 1 above)
Comment (Ivankivu) [reply UI/admin.html]: Thank you @mendozabree I am on it.

Comment (Ivankivu) [reply UI/admin.html]: @mendozabree I have pushed the new changes. Kindly please review the changes

File: UI/user.html (hunk 6 above)
Comment (Ivankivu) [reply UI/user.html]: yes it is!

File: UI/index.html (hunk 4 above)
Comment (Ivankivu) [reply UI/index.html]: cleared

File: UI/signin.html (hunk 5 above)
Comment (Ivankivu) [reply UI/signin.html:10]: cleared

File: UI/css/main.css (hunk 2 above)
Comment (Ivankivu) [reply UI/css/main.css]: cleared

**Review by mendozabree — APPROVED** [review]
LGTM

**Comment by Ivankivu** [comment]
@mendozabree Thank you for the guidance.

**Pull request merged**